### PR TITLE
Feature/images on s3

### PR DESCRIPTION
**Pull request by erik-sn**

### CODE REVIEW

File: src/components/profile/repositories/ImagePreview.js [hunk 1]
```diff
@@ -45,16 +45,18 @@ const styles = theme => ({
   },
 });
 
+const MAX_FILE_SIZE_bytes = 2000000;
+
 class ImagePreview extends Component {
   constructor(props) {
     super(props);
     let filePath = constants.SCIGYM_LOGO;
```
Comment (HendrikPN) [inline src/components/profile/repositories/ImagePreview.js:53]: `let filePath = constants.STATIC_URL + constants.SCIGYM_LOGO;`

File: src/components/profile/repositories/ImagePreview.js [hunk 2]
```diff
@@ -45,16 +45,18 @@ const styles = theme => ({
...
     if (props.avatar != null) {
-      filePath = props.avatar.uploadPath;
+      filePath = props.avatar.filePath;
```
Comment (HendrikPN) [inline src/components/profile/repositories/ImagePreview.js]: `filePath = constants.MEDIA_URL + props.avatar.filePath`

File: src/components/profile/repositories/ImagePreview.js [hunk 3]
```diff
@@ -45,16 +45,18 @@ const styles = theme => ({
...
     }
     this.state = {
       avatar: props.avatar,
-      avatarURL: constants.STATIC_URL.concat(filePath),
+      avatarURL: constants.MEDIA_URL.concat(filePath),
```
Comment (HendrikPN) [inline src/components/profile/repositories/ImagePreview.js]: `avatarURL: filePath,`

File: src/components/profile/repositories/ImagePreview.js [hunk 4]
```diff
@@ -66,11 +68,11 @@ class ImagePreview extends Component {
     if (prevProps.avatar !== this.props.avatar) {
       let filePath = constants.SCIGYM_LOGO;
```
Comment (HendrikPN) [inline src/components/profile/repositories/ImagePreview.js:69]: let filePath = constants.STATIC_URL + constants.SCIGYM_LOGO;

File: src/components/profile/repositories/ImagePreview.js [hunk 5]
```diff
@@ -66,11 +68,11 @@ class ImagePreview extends Component {
...
       if (this.props.avatar != null) {
-        filePath = this.props.avatar.uploadPath;
+        filePath = this.props.avatar.filePath;
```
Comment (HendrikPN) [inline src/components/profile/repositories/ImagePreview.js]: `filePath = constants.MEDIA_URL + this.props.avatar.filePath;`

File: src/components/profile/repositories/ImagePreview.js [hunk 6]
```diff
@@ -66,11 +68,11 @@ class ImagePreview extends Component {
...
       }
       this.setState({
         avatar: this.props.avatar,
-        avatarURL: constants.STATIC_URL.concat(filePath),
+        avatarURL: constants.MEDIA_URL.concat(filePath),
```
Comment (HendrikPN) [inline src/components/profile/repositories/ImagePreview.js:75]: `avatarURL: filePath,`

**Review by HendrikPN — CHANGES_REQUESTED** [review]
There seems to be an issue with getting the readme [play.md](https://github.com/HendrikPN/scigym-web/blob/master/src/components/get_started/Play.jsx#L56) and [contribute.md](https://github.com/HendrikPN/scigym-web/blob/master/src/components/get_started/Contribute.jsx#L56). We should probably add them to the statics files somehow and get them from `constants.js.

File: src/components/get_started/Contribute.jsx [hunk 7]
```diff
@@ -53,9 +56,9 @@ class Contribute extends Component {
     markdown: '',
   };
   componentDidMount() {
-    const mdPath = require('./contribute.md');
-
-    fetch(mdPath)
+    fetch(constants.CONTRIBUTE, {
```
Comment (HendrikPN) [inline src/components/get_started/Contribute.jsx]: This does not work! 
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://scigym.s3.eu-central-1.amazonaws.com/markdown/contribute.md. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).

File: src/components/get_started/Contribute.jsx [hunk 8]
```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
```
Comment (HendrikPN) [inline src/components/get_started/Contribute.jsx]: remove

File: src/components/get_started/Contribute.jsx [hunk 9]
```diff
@@ -1,12 +1,15 @@
...
 
 import { withStyles } from '@material-ui/core';
 import Typography from '@material-ui/core/Typography';
 import Card from '@material-ui/core/Card';
 import CardMedia from '@material-ui/core/CardMedia';
 
 import constants from '../../utils/constants';
+import api from '../../utils/api';
```
Comment (HendrikPN) [inline src/components/get_started/Contribute.jsx]: remove

File: src/components/get_started/Play.jsx [hunk 10]
```diff
@@ -53,9 +53,7 @@ class Play extends Component {
     markdown: '',
   };
   componentDidMount() {
-    const mdPath = require('./play.md');
-
-    fetch(mdPath)
+    fetch(constants.PLAY)
```
Comment (HendrikPN) [inline src/components/get_started/Play.jsx:56]: this does not work

**Review by HendrikPN — CHANGES_REQUESTED** [review]
There are some problems here that I cannot solve....

File: src/components/get_started/Contribute.jsx (hunk 8 above)
Comment (erik-sn) [reply src/components/get_started/Contribute.jsx]: I'm confused, your commit adds these...and this comment says to remove them?

Comment (HendrikPN) [reply src/components/get_started/Contribute.jsx]: Yeh. This is a "test" commit. It still had these redundant lines in them. I wanted you to see the problem. The only problem is the lines where I explicitly say "This does not work"

**Review by HendrikPN — APPROVED** [review]

**Pull request merged**